packages/moonstone/PictureInGraphics/PictureInGraphics.js

```diff
@@ -0,0 +1,163 @@
+	publicClassNames: ['pictureInGraphics', 'textOverlay']
+};
+
+class PictureInGraphicsBase extends React.Component {
+	static propTypes = {
+		/**
+		 * Any children `<source>` tag elements will
+		 * be sent directly to the `<video>` element as video sources.
+		 *
+		 * @type {Node}
+		 * @public
+		 */
+		source: PropTypes.node.isRequired,
+
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal Elements and states of this component.
+		 *
+		 * The following classes are supported:
+		 *
+		 * * `pictureInGraphics` - The root class name
+		 * * `textOverlay` - class name for `textOverlayContent`
+		 *
+		 * @type {Object}
+		 * @public
+		 */
+		css: PropTypes.object,
+
+		/**
+		 * Image path for image overlay.
+		 * When image is displayed, video is not displayed even though it is playing.
+		 *
+		 * @type {String}
+		 * @public
+		 */
+		imageOverlaySrc: PropTypes.string,
+
+		/**
+		 * Placeholder for image overlay
+		 *
+		 * @type {String}
+		 * @public
+		 */
+		placeholder: PropTypes.string,
+
+		/**
+		 * Text of text overlay
+		 *
+		 * @type {String}
+		 * @public
+		 */
+		textOverlayContent: PropTypes.string,
+
+		/**
+		 * Video component to use. The default (`'video'`) renders an `HTMLVideoElement`. Custom
+		 * video components must have a similar API structure, exposing the following APIs:
+		 *
+		 * Methods:
+		 * * `load()` - load video
+		 *
+		 * @type {Component}
+		 * @default 'video'
+		 * @public
+		 */
+		videoComponent: PropTypes.oneOfType([PropTypes.string, PropTypes.func])
+	}
+
+	static defaultProps = {
+		placeholder: defaultPlaceholder,
+		videoComponent: 'video'
+	}
+
+	constructor (props) {
+		super(props);
+		this.video = null;
+		this.renderStyles = cfgStyles ? styles(cfgStyles) : false;
+	}
+
+	componentDidUpdate (prevProps) {
+		const {source} = this.props;
+		const {source: prevSource} = prevProps;
+
+		if (!compareSources(source, prevSource)) {
+			this.video.load();
+		}
+	}
+
+	setVideoRef = (video) => {
+		this.video = video;
+	}
+
+	render () {
+		let p = Object.assign({}, this.props);
+		if (this.renderStyles) p = this.renderStyles(p);
+
+		const {
+			className,
+			css,
+			source,
+			placeholder,
+			imageOverlaySrc,
+			textOverlayContent,
+			videoComponent,
+			...rest} = p;
+		const containerClassName = classNames(className, css.pictureInGraphics);
+
+		return (
+			<div {...rest} className={containerClassName} >
+				<Media
+					autoPlay
+					className={css.video}
+					component={videoComponent}
+					controls={false}
+					ref={this.setVideoRef}
+				>
+					{source}
+				</Media>
+				{imageOverlaySrc ? <Image className={css.image} placeholder={placeholder} sizing="fit" src={imageOverlaySrc} /> : null}
+				{textOverlayContent ? (<Marquee alignment="center" className={css.textOverlay} marqueeOn="render">{textOverlayContent}</Marquee>) : null}
+			</div>
+		);
+	}
+}
+
+const PictureInGraphics = Pure(
```

Comment:
Let's use `ramda/src/compose` and call this `PictureInGraphicsDecorator` to be consistant with other components.
You should assign `const PictureInGraphics = PictureInGraphicsDecorator(PictureInGraphicsBase)` and export it as default.